app/services/mapping/AuthorisationHoldersBuilder.scala

```diff
@@ -33,8 +33,9 @@ import wco.datamodel.wco.declaration_ds.dms._2.{
 
 import scala.collection.JavaConverters._
 
-class AuthorisationHoldersBuilder @Inject()() extends ModifyingBuilder[Declaration] {
```

Comment:
Whys the trait gone?

Reply:
This was a rebase issue I had on Friday. looks like I resolved this file incorrectly will change it back